dotCMS/src/integration-test/java/com/dotcms/concurrent/DotConcurrentFactoryTest.java

```diff
@@ -101,4 +103,44 @@ public void run() {
         }
 
     }
+    
+    
+    
+    
+    /**
+     * This tests that the delayed queue will run jobs in the future, efficiently
+     * 
+     * @throws Exception
+     */
+    
+    @Test
+    public void test_delayed_queue() throws Exception{
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5775864)